editor/src/clj/internal/util.clj

```diff
@@ -546,3 +558,94 @@
    (deep-keep-kv-helper identity value-fn nil value))
   ([finalize-coll-value-fn value-fn value]
    (deep-keep-kv-helper finalize-coll-value-fn value-fn nil value)))
+
+(defn into-multiple
+  "Like core.into, but transfers the input into multiple collections in a single
+  pass. Instead of a single destination and optional transducer, takes a seq of
+  destinations and a seq of transducers. The number of destinations must match
+  the number of transducers. Returns a seq of the populated destinations."
```

Comment:
Messy to write and has optimized implementations depending on the number of destinations, but I think it's worth it for the significant speed-up when we can use it.

Reply:
Is there a significant speed-up? You iterate once, but you also create a lot of intermediate map entries...